pyaedt/modeler/Object3d.py

```diff
@@ -674,16 +703,27 @@ def surface_material_name(self):
 
     @property
     def group_name(self):
-        """Get the group to which an object belongs
+        """Group the object belongs to.
+        
+        Returns
+        -------
+        str
+            Name of the group.
+        
         """
         if 'Group' in self.valid_properties:
             self.m_groupName = retry_ntimes(10, self.m_Editor.GetPropertyValue, "Geometry3DAttributeTab", self._m_name, 'Group')
             return self.m_groupName
 
     @property
     def material_name(self):
-        """Get or set the material name of the object
-        If the material does not exist, then send a warning and do nothing
+        """Material name of the object.
+        
+        Returns
+        -------
+        str or ``None``
```

Comment:
```suggestion
        str or None
```

Reason why it's simply None is because sphinx is smart enough to know that this will be a type.